cmake/external/mkldnn.cmake

```diff
@@ -1,41 +1,28 @@
 include (ExternalProject)
 
 set(MKLDNN_URL https://github.com/intel/mkl-dnn.git)
-# If MKLDNN_TAG is updated, check if MKLML_VERSION and platform.cmake.patch need to be updated.
-set(MKLDNN_TAG v0.18.1)
-set(MKLML_VERSION 2019.0.5.20190502)
+set(MKLDNN_TAG v1.0.2)
 
 if(WIN32)
-  set(MKLML_OS_VERSION_STR "win")
-  set(MKLML_FILE_EXTENSION "zip")
   set(MKLDNN_SHARED_LIB mkldnn.dll)
   set(MKLDNN_IMPORT_LIB mkldnn.lib)
   if(onnxruntime_USE_MKLML)
-    # Windows-only updated MKLML binary which contains fix for thread cleanup hang.
-    set(MKLML_VERSION 2020.0.20190813)
-    set(MKLML_SHARED_LIB mklml.dll)
-    set(MKLML_IMPORT_LIB mklml.lib)
     set(IOMP5MD_SHARED_LIB libiomp5md.dll)
     set(IOMP5MD_IMPORT_LIB libiomp5md.lib)
   endif()
 else()
-  set(MKLML_FILE_EXTENSION "tgz")
   if (APPLE)
     set(MKLDNN_SHARED_LIB libmkldnn.0.dylib)
-    set(MKLML_OS_VERSION_STR "mac")
   else()
     set(MKLDNN_SHARED_LIB libmkldnn.so.0)
-    set(MKLML_OS_VERSION_STR "lnx")
   endif()
   if(onnxruntime_USE_MKLML)
-    set(MKLML_SHARED_LIB libmklml_intel.so)
     set(IOMP5MD_SHARED_LIB libiomp5.so)
   endif()
 endif()
 
 if (onnxruntime_USE_MKLML)
-  set(MKLDNN_VERSION_SHORT v0.20)
-  set(MKLML_URL https://github.com/intel/mkl-dnn/releases/download/${MKLDNN_VERSION_SHORT}/mklml_${MKLML_OS_VERSION_STR}_${MKLML_VERSION}.${MKLML_FILE_EXTENSION})
```

Comment:
don't remove this. make sure --use_mklml still works (by itself, without --use_mkldnn)